packages/next-urql/package.json

```diff
@@ -42,6 +42,7 @@
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
     "graphql": "^15.4.0",
+    "next": "^10.0.3",
```

Comment:
I think there's a PR that we could basically revert to get here. However, the original intention was to make sure that we don't need the `next` package or depend on it as it literally comes with hundreds of its own dependencies for builds and etc some conflicting with our own 😅

We could look into why the generic isn't working as intended